package.json

```diff
@@ -36,7 +36,7 @@
     }
   ],
   "dependencies": {
-    "react-native-tabs": "^1.0.2"
+    "react-native-tab-navigator": "joenoon/react-native-tab-navigator#master"
```

Comment:
Only potential concern would be if they list react-native-tab-navigator in their package.json separately. Maybe bump the version number in your master branch to avoid conflict?